scripts/qa_crowbarsetup.sh

```diff
@@ -4608,7 +4608,10 @@ function onadmin_prepare_crowbar_upgrade()
         if crowbarctl upgrade repocheck crowbar --format plain | grep "missing" ; then
             complain 11 "Some repository is missing on admin server. Cannot upgrade."
         fi
-        safely crowbarctl upgrade prepare
+        # FIXME: crowbarctl command can time out easily
```

Comment:
I don't want to delay the discussion further, but what I don't understand is that `crowbar_api_request` has a timeout of 300s as well, so why should this not work, or work any different than `crowbarctl`? if it really doesn't work then it's likely a bug in `crowbarctl`